guide/02-api-overview/release-notes.ipynb

```diff
@@ -1,5 +1,517 @@
 {
```

Comment:
<p>"Adds support (and documentation) for two Enumerations to use as input for various functions. See code snippet at"  -&gt; "Simplifies how units are specified by introducing two Enumerations. See code snippet for <code>area_and_lengths</code>:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1101/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>Added this revision!</p>